src/test/java/seedu/realodex/model/person/BirthdayTest.java

```diff
@@ -32,9 +33,16 @@ public void isValidBirthday() {
         assertFalse(Birthday.isValidBirthday("29Feb2023")); // not a leap year
         assertFalse(Birthday.isValidBirthday("1-jan-2001")); // not supposed to have '-'
 
-        // valid email
+        // valid date
         assertTrue(Birthday.isValidBirthday("29Feb2024")); // leap year
         assertTrue(Birthday.isValidBirthday("12May2003"));
+        assertTrue(Birthday.isValidBirthday("08Aug1888"));
+
+        // invalid dates
+        assertFalse(Birthday.isValidBirthday("01May2009233"));
+        assertFalse(Birthday.isValidBirthday("0"));
+        assertFalse(Birthday.isValidBirthday("1jan2032")); //cant be in future
```

Comment:
Good test (we might need to update this when Realodex succeeds in the next 10 years)

Reply:
# It will